loomi/loomi-ui-flutter
# feature/new-components-and-fix

_Changes:_

_1. What i did:_

- Create Custom Quantity Selector Widget
- Fix Custom Number Counter Widget
- Create Custom Text Badge Widget
- Create Custom Alert Widget
- Fix Custom Check Box Widget

_2. What to test:_

- [ ] Layout on different devices
- [ ] Happy path

---

_How to get there_

_1. Flavor and login informations_

> _flavor_: staging

_2. Steps_

- Go to Example Screen
- Check the Widgets described

---

_Observations:_

_1. Known issues_

- Put them here if there are any

_2. Code review pratices_

- Unecessary prints/log (ctrl+f -> [ print( ] )
- Unecessary comments (ctrl+f -> [ // ] )
- Code organization
- Naming of functions and variables
- Repeated code
